<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChineseChess</class>
 <widget class="QMainWindow" name="ChineseChess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChineseChess</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRestart">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重新开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/img/images/baby.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="opponentLaber">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>黑方</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbRobot">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>人机对战</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">playChoose</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="rbHuman">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>人人对战</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">playChoose</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="playChoose"/>
 </buttongroups>
</ui>
